package.json

```diff
@@ -1,45 +1,49 @@
 {
-  "public": true,
```

Comment:
Perhaps we should copy over the [`.editorconfig`](https://github.com/preactjs/preact/blob/master/.editorconfig)?

Reply:
bleh, yeah. I used github.dev and it told me this was the right spacing haha

Reply:
fixed for now